### PR TITLE
[DUOS-2905] Supports external & open access property in the data library table

### PR DESCRIPTION
**Pull request by PintoGideon**
### Addresses

https://broadworkbench.atlassian.net/jira/software/c/projects/DUOS/boards/123?selectedIssue=DUOS-2905

### Summary

Supports external & open access property in the data library table

----
Have you read [Terra's Contributing Guide](https://github.com/DataBiosphere/terra-ui/wiki/Contributor-Guide) lately? If not, do that first.

- Label PR with a Jira ticket number and include a link to the ticket
- Label PR with a security risk modifier [no, low, medium, high]
- PR describes scope of changes
- Get a minimum of one thumbs worth of review, preferably two if enough team members are available
- Get PO sign-off for all non-trivial UI or workflow changes
- Verify all tests go green
- Test this change deployed correctly and works on dev environment after deployment

### CODE REVIEW

**Comment by PintoGideon** [comment]
@fboulnois  I implemented your previous feedback in this PR to keep the diff cleaner to review. Let me know what you think.

File: src/components/data_search/DatasetSearchTable.js [hunk 1]
```diff
@@ -296,7 +296,17 @@ export const DatasetSearchTable = (props) => {
                     value: dataset.participantCount,
                   },
                   {
-                    value: dataset.dac?.dacEmail ? <Link href={'mailto:' + dataset.dac.dacEmail}>{dataset.dac?.dacName}</Link> : dataset.dac?.dacName,
+                    value:() => {
```
Comment (fboulnois) [inline src/components/data_search/DatasetSearchTable.js]:
```suggestion
                    value: () => {
```

tiny nit

**Review by fboulnois — APPROVED** [review]
Looks good! One small suggestion:

**Review by fboulnois — COMMENTED** [review]
Also, I often use `git rebase` to clean up my PRs, let me know if you'd like help in the future on using this.

File: src/components/data_search/DatasetSearchTable.js [hunk 2]
```diff
@@ -30,8 +30,8 @@ const datasetTableHeader = [
   'Data Use',
   'Data Types',
   'Participants',
-  'Data Location',
   'DAC',
```
Comment (aarohinadkarni) [inline src/components/data_search/DatasetSearchTable.js:33]: Just checking the ticket, should the 'DAC' header be changed to 'Access Type'?
```suggestion
  'Access Type',
```

**Review by aarohinadkarni — APPROVED** [review]
looks good besides one minor comment 👍

**Comment by PintoGideon** [comment]
@fboulnois  Yes would love to know your git and development workflow.

**Pull request closed without merging**